catlas/filters.py

```diff
@@ -98,10 +100,12 @@ def bulk_filter(config, dask_df, sankey, initial_bulks):
                     )
                 ]
 
-            elif name == "filter_by_bulk_e_above_hull":
+            elif (
+                name == "filter_by_bulk_e_above_hull" and "bulk_e_above_hull" in columns
+            ):
                 dask_df = dask_df[dask_df.bulk_e_above_hull <= val]
 
-            elif name == "filter_by_bulk_band_gap":
+            elif name == "filter_by_bulk_band_gap" and "bulk_band_gap" in columns:
```

Comment:
Same

Reply:
samr

Reply:
original question was 'why is this necessary'?

Reply:
yeah the reply is the same for all so I figured we could collapse into one thread

Reply:
oh I totally lost the original thread- was it resolved?

Reply:
github hid the original thread